Claroty/claroty-xdome/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "value": "{event.reason}",
+    "conditions": [
+      {
+        "field": "event.reason"
+      }
+    ]
```

Comment:
Could you add one smart-description per event.category and event.type?
ie one smart-descriptions for network connection events and one for authentication events.